docs/docs-site/content/administrator/configuration/connectors/_index.md

```diff
@@ -803,6 +803,21 @@ Alternative:
     ## The JDBC driver clickhouse-jdbc.jar and its related jars need to be in the CLASSPATH environment variable.
     options='{"url": "jdbc:clickhouse://localhost:8123", "driver": "ru.yandex.clickhouse.ClickHouseDriver", "user": "readonly", "password": ""}'
 
+
+### Apache Kyuubi
+
+[Apache Kyuubi](https://kyuubi.readthedocs.io/en/master/) is a distributed and multi-tenant gateway to provide serverless SQL on data warehouses and lakehouses.
+
+Give Hue the information about the database source:
+
+    [[[kyuubi]]]
+    name=Kyuubi
+    interface=jdbc
+    ## Specific options for connecting to the Kyuubi server.
+    ## The JDBC driver kyuubi-hive-jdbc.jar and its related jars need to be in the CLASSPATH environment variable.
```

Comment:
it's recommended to use [kyuubi-hive-jdbc-shaded](https://mvnrepository.com/artifact/org.apache.kyuubi/kyuubi-hive-jdbc-shaded) over kyuubi-hive-jdbc, the shaded is a single jar without additional dependencies